### PR TITLE
docs: Added AWS GUI testing instructions.

### PR DESCRIPTION
**Pull request by AndrewDorn**
## Summary of Changes

Addresses [ORCA-585: Test/document method of accessing GraphQL UI that is running in AWS](https://bugs.earthdata.nasa.gov/browse/ORCA-585)

## Changes

* Added docs on testing GQL UI in AWS

## Type of change

- [x] This change requires a documentation update

## How Has This Been Tested?

Run in AWS.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [x] Testing documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
    - [x] Manual tests (outlined above)
- [x] New and existing unit tests pass locally with my changes
    - [x] Automated tests passing (if not fork)
- [x] Any dependent changes have been merged and published in downstream modules
- [x] My code has passed security scanning
    - [x] Snyk
    - [x] git-secrets

### CODE REVIEW

**Review by Quarriers89 — COMMENTED** [review]
Hey, Andrew. Looks OK to me.

File: graphql/README.md [hunk 1]
```diff
@@ -40,6 +41,34 @@ Visit the [Developer Guide](https://nasa.github.io/cumulus-orca/docs/developer/d
...
+   ```json
+   {
+     "name": "ORCA_ENV",
+     "value": "development"
+   }
+   ```
+   then redeploy.
+   
+2. Run the following bash command, 
+   replacing `i-00000000000000000` with your ec2 instance name.
```
Comment (rizbihassan) [inline graphql/README.md:58]: How do we know which instance? Is it the Cumulus deployed EC2 instance ID named as named `<prefix>-Cumulus-ECSCluster`?

Might be worth adding this to get the ID from CLI
  ```bash
  aws ec2 describe-instances --filters Name=instance-state-name,Values=running Name=tag:Name,Values=<PREFIX>-CumulusECSCluster --query "Reservations[*].Instances[*].InstanceId" --output text
  ```

**Review by rizbihassan — CHANGES_REQUESTED** [review]

File: graphql/README.md [hunk 2]
```diff
@@ -40,6 +41,34 @@ Visit the [Developer Guide](https://nasa.github.io/cumulus-orca/docs/developer/d
   - Use the command `docker run -d -p 5000:5000 --env-file path/to/env imageName` 
     - Replace `imageName` with the name of your built image.
     - Replace `path/to/env` with the path to the env file you created in the previous step.
+- The GraphQL URL is `http://0.0.0.0:5000/graphql`
```
[the remark this answers is not in the document]
Comment (AndrewDorn) [reply graphql/README.md]: Looks like that works too. Simplified.

File: graphql/README.md [hunk 3]
```diff
@@ -40,6 +41,34 @@ Visit the [Developer Guide](https://nasa.github.io/cumulus-orca/docs/developer/d
...
+- 
+### AWS Testing
+1. If you wish to enable the developer GUI, add/modify the "ORCA_ENV" 
+   in the `environment` section in your Terraform's `aws_ecs_task_definition` to
+   ```json
+   {
+     "name": "ORCA_ENV",
+     "value": "development"
```
[the remark this answers is not in the document]
Comment (AndrewDorn) [reply graphql/README.md:52]: Environment variables are key-value-pairs. The string `${local.graphql_port}` is only used for the KVP with `"name": "PORT"`

[the remark this answers is not in the document]
Comment (AndrewDorn) [reply graphql/README.md:52]: See the KVP I added with `"name": "ORCA_ENV"`

File: graphql/README.md (hunk 1 above)
Comment (AndrewDorn) [reply graphql/README.md]: Added.

File: graphql/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (AndrewDorn) [reply graphql/README.md:52]: Added verbiage to indicate that this is an inclusion, rather than a total replacement.

**Review by rizbihassan — APPROVED** [review]
looks good. It will be interesting to see the UI running in aws in a future orcatecture meeting.

**Pull request merged**